src/usb_moded-configfs.c

```diff
@@ -188,6 +191,13 @@ static void configfs_read_configuration(void)
                         temp_setting);
     g_free(temp_setting);
 
+    temp_setting = configfs_get_conf("gadget_udc_device",
+                              DEFAULT_GADGET_UDC_DEVICE);
+    GADGET_UDC_DEVICE =
+        g_strdup_printf("%s",
```

Comment:
[I thought that I commented about this already but can't see it anymore... another attempt]

As this is just the config string without further formatting, it could be simplified to:

    GADGET_UDC_DEVICE = configfs_get_conf(...)